.github/workflows/app-build-verify.yml

```diff
@@ -127,7 +131,8 @@ jobs:
       if: endsWith(inputs.repository, 'briefcase')
       uses: actions/download-artifact@v4.1.7
       with:
-        name: packages-briefcase
+        pattern: ${{ format('{0}*', inputs.briefcase-artifact-name) }}
+        merge-multiple: true
```

Comment:
It's not clear to me why this needs to be parameterised. AFAICT, it's only used by the briefcase repo, and will only ever be set to the name of the briefcase package. What do we gain by parameterising the name of the briefcase artefact?

Reply:
Mostly because the name of the artifact is arbitrary. We can hardcode it here but that just means this workflow is dependent on however the package is uploaded as an artifact. Is that what you mean?

Reply:
> Mostly because the name of the artifact is arbitrary. 

Well... yes and no. Yes. the artefact *could* be named anything... but briefcase only ever uses one name, and this is only ever used by Briefcase. Unless we're anticipating Briefcase regularly changing its artefact name, I'm not sure what we gain by parameterization of the briefcase artefact name.

Reply:
(to be clear - this is a case where parameterisation definitely isn't *wrong* - I'm just wary that there's already a lot of moving parts and interactions between obscure parts of the Actions API, so I'm keen to keep this as simple as humanly possible)

Reply:
That's fair; if the artifact name changes, it'll be obvious to change it here.